arxiv/base/static/sass/bulma/README.md

```diff
@@ -0,0 +1,106 @@
+# [Bulma](https://bulma.io)
```

Comment:
Should this be in here?

Reply:
I'm not sure -- advise me on GitHub manners?  Is it better to have the entirety of the bulma package intact, as authored, or remove the readme and changelog so that the files don't accidentally get picked up by any of our documentation?
I'm good either way. Suppose that separating the Bulma files into their own directory is enough separation to indicate their collective function as a library.